src/mbed_tools/build/flash.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Flash binary onto the connected device."""
+
+import shutil
+import os
+from os.path import basename, exists
```

Comment:
We shouldn't need these dependencies as we're using `pathlib`.

Reply:
Fixed